api/lib/rest-to-tx-converter.js

```diff
@@ -20,27 +20,18 @@ function RestToTxConverter() {}
  */
 RestToTxConverter.prototype.convert = function(payment, callback) {
   function isSendMaxRequired() {
```

Comment:
I think it would be technically more correct to negate the returns and call this function `isSendMaxAllowed` now.